### DISCUSSION TITLE
Is it possible to install ipython with nteract in one installer?

### DISCUSSION THREAD

**Discussion by pplonski** [nteract/archived-desktop-app#5535] [Q&A] [answered]
Thank you for amazing app! :) Got a question.

I'm using macOS.  I downloaded nteract and after install there is an error message that kernel is missing. Is it possible to have all needed packages installed with one download? I want to use Python kernel (only). This will make installing / using nteract simpler.

**Comment by captainsafia** [comment, marked as answer]
It certainly is! 

This is something we've been tracking for a while in https://github.com/nteract/nteract/issues/3030.
